crates/rover-client/src/query/partial/push.rs

```diff
@@ -31,7 +31,7 @@ pub fn run(
 ) -> Result<PushPartialSchemaResponse, RoverClientError> {
     let data = client.post::<PushPartialSchemaMutation>(variables)?;
     let push_response = get_push_response_from_data(data)?;
-    build_response(push_response)
```

Comment:
I assume the change here is because clippy knows tha t `build_response` can't actually fail?

Reply:
yup!